### DISCUSSION TITLE
pg_auto_parameterize and default_import_slice behavior

### DISCUSSION THREAD

**Discussion by davekaro** [jeremyevans/sequel#2241] [General]
When the pg_auto_parameterize extension is used, it limits all inserts to 40 at a time, even when using the `no_auto_parameterize` option. I'd like to import more than 40 at once, in this case I have no need to parameterize the values as they are already coming straight from the DB and are known safe. Maybe just changing the extension like below?

```
def default_import_slice
  @opts[:no_auto_parameterize] ? super : 40
end
```

**Comment by jeremyevans** [comment]
Thank you for point that out. Your suggested change makes sense to me.  Would you like to submit a pull request for that?  If not, I can make the change.

**Reply by davekaro** [reply]
Sure thing - see https://github.com/jeremyevans/sequel/pull/2242
